src/Illuminate/View/Compilers/Concerns/CompilesHelpers.php

```diff
@@ -24,4 +24,15 @@ protected function compileMethod($method)
     {
         return "<?php echo method_field{$method}; ?>";
     }
+
+    /*
+     * Compile the dd statements into valid PHP.
+     *
+     * @param  string  $args
+     * @return string
+     */
+    protected function compileDd($args)
+    {
+        return "<?php dd{$args}; ?>";
```

Comment:
Would not be it good if we make it like `<?php call_user_func_array('dd', $args); ?>`?

Reply:
Better to have `dd(...$args)` actually.

Reply:
I think because of `$args` is a string, that olso contains parentheses symbols.

Reply:
`$args` will not be a string always. :)

Reply:
This is literally making a string containing the `dd()` function - not actually running the function. So you cant do things like `dd(...$args)` - because that turns `@dd($var1, $var2)` into `<?php dd(...($var1, $var)); ?>`

It works perfectly as is (I've tested it against live code).

Reply:
Okay, 👍  Then please make changes in the doc block that `$args` param is an array... It will pass an array and compiler will convert it into `<?php dd(...$args); ?>`

Reply:
@Modelizer - its not an array. It's a string. That's what Blade passes and generates.

Reply:
then please make it `$arg` it's making me crazy because `$args` is always use for an array 😅